src/dev.c

```diff
@@ -33,6 +33,18 @@ static int match_usbdev(const struct usb_device_info *devinfo);
 
 static struct device *dev_list = NULL;
 
+static int device_evt_num(const char* path)
+{
+	const char needle[] = "/dev/input/event";
+	char* pos = strstr(path, needle);
+
+	if (pos == NULL) return 0;
+
+	pos += strlen(needle);
+
+	return (int)strtol(pos, NULL, 10);
```

Comment:
If you are not doing non-decimal conversions, and you're not using the end pointer, you might as well just use atoi here. Also lose the cast.